### PR TITLE
DOCS-724: Fixes broke xrefs

### PR DESCRIPTION
**Pull request by ctauchen**
https://tigera.atlassian.net/browse/DOCS-724

Removes broken cross references from the docs. 

By and large, these were caused by moving CE 3.15 docs into CC. Some links were to files not present in CC.

### CODE REVIEW

**Comment by ctauchen** [comment]
@doucol Could you take a look at the two components I've modified here, both called EnvironmentFile.js?
* https://github.com/tigera/docs/pull/384/files#diff-d27a8a7471362addc88efe2bad7e2b25a82248737d2bd8276b6ff119cbb28cc4
* https://github.com/tigera/docs/pull/384/files#diff-33d3e6cf23e9caa367f521992c1405bed0090d74d5094a1e55bba0a5622fab77

File: calico-cloud/_includes/components/EnvironmentFile.js [hunk 1]
```diff
@@ -10,15 +10,16 @@ export default function EnvironmentFile(props) {
...
-          host. For more help, see the{' '}
-        </span>
         {props.install === 'container' ? (
-          <Link href={`${baseUrl}/reference/component-resources/node/configuration`}>{props.nodecontainer} configuration reference</Link>
-        ) : (
-          <Link href={`${baseUrl}/reference/felix/configuration`}>Felix configuration reference</Link>
-        )}
+            <span>
+               Use the following guidelines and sample file to define the environment variables for starting Calico on the
+          host. For more help, see the <Link href={`${baseUrl}/reference/component-resources/node/configuration`}>{props.nodecontainer} configuration reference</Link>
```
Comment (doucol) [inline calico-cloud/_includes/components/EnvironmentFile.js]: The formatting / readability could be improved with aligned indention for the text under the span.

File: calico-cloud_versioned_docs/version-3.15/_includes/components/EnvironmentFile.js [hunk 2]
```diff
@@ -9,17 +9,18 @@ import { baseUrl } from '../../variables';
...
-          <Link href={`${baseUrl}/reference/component-resources/node/configuration`}>{props.nodecontainer} configuration reference</Link>
-        ) : (
-          <Link href={`${baseUrl}/reference/felix/configuration`}>Felix configuration reference</Link>
-        )}
-      </p>
+        <p>
+            {props.install === 'container' ? (
+                <span>
+               Use the following guidelines and sample file to define the environment variables for starting Calico on the
+          host. For more help, see the <Link href={`${baseUrl}/reference/component-resources/node/configuration`}>{props.nodecontainer} configuration reference</Link>
```
Comment (doucol) [inline calico-cloud_versioned_docs/version-3.15/_includes/components/EnvironmentFile.js]: Again, formatting / readability  - but otherwise looks fine.

**Review by doucol — CHANGES_REQUESTED** [review]
Just minor formatting request for readability.

File: calico-cloud/_includes/components/EnvironmentFile.js (hunk 1 above)
Comment (ctauchen) [reply calico-cloud/_includes/components/EnvironmentFile.js]: Thanks, done!

File: calico-cloud_versioned_docs/version-3.15/_includes/components/EnvironmentFile.js (hunk 2 above)
Comment (ctauchen) [reply calico-cloud_versioned_docs/version-3.15/_includes/components/EnvironmentFile.js]: Done!

**Pull request merged**